core/modules/system/system.menu.inc

```diff
@@ -101,6 +102,14 @@ function system_menu_block_form($config) {
       ),
     ),
   );
+  if ($config['menu_name'] == 'main-menu') {
+    $form['menu_toggle'] = array(
+      '#type' => 'checkbox',
+      '#title' => 'Display menu toggle button on small screens',
+      '#default_value' => $config['menu_toggle'],
+      '#description' => t('When enabled, a menu toggle button—commonly known as a "hamburger" icon—will appear on small screens and allow the user to toggle the visibility of the primary navigation.'),
```

Comment:
Can we change this to `When enabled, a menu icon, commonly known as a "hamburger" icon, will appear on small screens to allow the menu to be expanded and collapsed.`